org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XtypeIdeModule.java

```diff
@@ -1,14 +1,14 @@
 /*******************************************************************************
- * Copyright (c) 2010, 2017 itemis AG (http://www.itemis.eu) and others.
+ * Copyright (c) 2010, 2020 itemis AG (http://www.itemis.eu) and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  *******************************************************************************/
- package org.eclipse.xtext.xbase.ide
+ package org.eclipse.xtext.xbase.ide;
 
 /** 
  * Use this class to register ide components.
  */
-class XbaseIdeModule extends AbstractXbaseIdeModule {
+public class XtypeIdeModule extends AbstractXtypeIdeModule {
```

Comment:
Although this diff looks a little bit weird, I think it should be fine.

Reply:
diff broken?????

Reply:
Since is was no real git mv, the file rename / diff stuff is based on heuristics.